### PR TITLE
Minimize all panels on user interaction on the map

### PR DESCRIPTION
**Pull request by bbecquet**
## Description
Automatically minimize all the mobile panels when the user interacts with the map, like we already do with the POI panel.

## Why
Give the user the maximum screen space when they want to interact with the map.

### CODE REVIEW

**Review by G-Ray — DISMISSED** [review]
:+1: Easy to review and nice UX improvement

**Comment by bbecquet** [comment]
I've amended the commit to remove the exception for the DirectionPanel, after the design confirmed it's ok to have the same behavior for the route result panel.

File: src/panel/PanelManager.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default class PanelManager extends React.Component {
     window.times.appRendered = Date.now();
 
     listen('map_user_interaction', () => {
-      if (this.state.ActivePanel === PoiPanel && isMobileDevice()) {
+      if (isMobileDevice()) {
```
Comment (G-Ray) [inline src/panel/PanelManager.jsx:54]: I guess it's the wanted behavior even in direction panel ?

Comment (bbecquet) [reply src/panel/PanelManager.jsx:54]: Yep, that's what I explained in the comment :)

Comment (G-Ray) [reply src/panel/PanelManager.jsx:54]: ok, I missed it when re-reviewing

**Review by G-Ray — APPROVED** [review]

**Pull request merged**